cdap-docs/_common/_themes/cdap/layout.html

```diff
@@ -117,11 +117,6 @@
     })(window,document,'script','dataLayer','{{ html_google_tag_manager_code }}');</script>
     {%- endif %}
     <!-- Google Tag Manager end -->
-    
-    <!-- Amazon SQS Page Usage Logging start -->
-    <script type="text/javascript" src="https://sdk.amazonaws.com/js/aws-sdk-2.1.35.min.js"></script>
-    <script type="text/javascript" src="http://docs.cask.co/resources/scripts/aws-sqs-1.0.0.js"></script>
```

Comment:
Do you mean, delete the file ``aws-sqs-1.0.0.js``? No, because we have a zillion files that reference that file. Instead, that file has been replaced with an empty stub.